api/application/client.go

```diff
@@ -621,6 +621,8 @@ func (c *Client) DestroyConsumedApplication(saasNames ...string) ([]params.Error
 		index = append(index, i)
 		args.Applications = append(args.Applications, params.DestroyConsumedApplicationParams{
 			ApplicationTag: names.NewApplicationTag(name).String(),
+			Force: force,
```

Comment:
This will need to be facade version aware since older facades would not support force, etc..